fetch-content.sh

```diff
@@ -3,20 +3,36 @@
 # Die on any error for Travis CI to automatically retry:
 set -e
 
+function download_url() {
+  URL="$1"
+  LOCAL_FILE="${1##*/}"
+  if [ -x "$(type -p wget)" ]; then
```

Comment:
Use `type -P` instead, because it is more reliable.

ref: `type` docs https://www.gnu.org/software/bash/manual/html_node/Bash-Builtins.html

>If the `-p` option is used, `type` either returns the name of the disk file that would be executed, or nothing if `-t` would not return `'file'`.
>
>The `-P` option forces a path search for each _name_, even if `-t` would not return `'file'`.

Reply:
I assume you don't want this merged yet then.

Reply:
Doesn't really matter to me.  If you wanted to merge it then you know what to chamge.